dev-qt/qtermwidget/qtermwidget-0.14.1.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A terminal emulator widget for Qt5"
+DESCRIPTION="QTermWidget is an open-source project originally \
+based on the KDE4 Konsole application, but it took its own direction later on \
+The main goal of this project is to provide a unicode-enabled \
+embeddable Qt widget for using as a built-in console or \
+terminal emulation widget it is compatible with BSD, Linux and OS X."
+HOMEPAGE="https://github.com/lxqt/qtermwidget/"
+COPYRIGHT="Petr Vanek"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/lxqt/qtermwidget/releases/download/$portVersion/qtermwidget-$portVersion.tar.xz"
+CHECKSUM_SHA256="84739f91e6ac5900a39ed7cbb254397a9428b172ee3fe0d1b6c827b751dc3b6c"
+PATCHES="qtermwidget-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qtermwidget$secondaryArchSuffix = $portVersion
+	lib:libqtermwidget5$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	qtermwidget${secondaryArchSuffix}_devel = $portVersion
+	devel:libqtermwidget5$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	qtermwidget$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	lxqt_build_tools$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	cmake .. $cmakeDirArgs \
+		-DCMAKE_BUILD_TYPE=Release \
```

Comment:
Superfluous "\" at the end.

Reply:
And why os there an \ at the end pf line 59?